src/cityreader/cityreader.py

```diff
@@ -16,26 +25,33 @@
 # should not be loaded into a City object.
 cities = []
 
+
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
+    # TODO Implement the functionality to read from the 'cities.csv' file
+    # For each city record, create a new City instance and add it to the
+    # `cities` list
+    with open('cities.csv', newline='') as csvfile:
+        cityreader = csv.reader(csvfile, delimiter=',')
+        for row in cityreader:
+            cities.append(City(f'{row[0]}, {row[1]}', row[3], row[4]))
     return cities
 
+
 cityreader(cities)
 
 # Print the list of cities (name, lat, lon), 1 record per line.
 for c in cities:
-    print(c)
```

Comment:
Second - this just wanted you to print a record per line. There are a couple of ways to do this. You could put all the things you wanted printed in the same print statement.
`print(c.name, c.lat, c.lon)`

or you could do `print(c)`. However, for this to work you would have needed to include a repr in the City class. 
`def __repr__(self):
      return "<{} {},{}>".format(self.name, self.lat, self.lon)`
Without the repr, if you just did print(c) you would have gotten a bunch of object printed out (which I'm sure you saw).